<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgTimeTests</class>
 <widget class="QDialog" name="dlgTimeTests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timing tests</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Number of Threads</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="togThread0">
        <property name="text">
         <string>GPU (if available)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="togThread1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="togThread4">
        <property name="text">
         <string>4 (max in Laptop)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="togThreadM">
        <property name="text">
         <string>Max of CPU</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>HKLmax</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="togHKL2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="togHKL5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="togHKL4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="togHKL3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="inpHKL5">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="inpHKL4">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="inpHKL3">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="inpHKL2">
        <property name="minimum">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="butStart">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="grpQuadrants">
     <property name="title">
      <string>Quadrants</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="togQ1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="togQ2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="togQ4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="inpSaveFilename"/>
     </item>
     <item>
      <widget class="QPushButton" name="butSaveFilename">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="lblComment">
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLineEdit" name="inpComment"/>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QCheckBox" name="togEnaUpdates">
     <property name="toolTip">
      <string>This may slow down if used via network</string>
     </property>
     <property name="text">
      <string>Enable GUI-Updates during calculation</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QCheckBox" name="togSaveImages">
     <property name="text">
      <string>Save images near the result file</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="butCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="inpNumLoops">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblNumCalc">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of calculations:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number of repetitions:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="togSaveFile">
     <property name="text">
      <string>save result into file:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
